src/main/kotlin/test/ktortemplate/Application.kt

```diff
@@ -76,8 +77,10 @@ fun Application.module(configOverrides: ApplicationConfig? = null) {
     }
 
     install(Health) {
+        val readinessDbTimeout: ApplicationConfigValue? = environment.config
```

Comment:
you may be able to simply declare 
val readinessDbTimeout = environment.config.propertyOrNull("healthcheck.readiness.database.timeoutMillis")

The type system should be able to infer the correct type

Reply:
no, propertyOrNull returns an ApplicationConfigValue, so there's no possible automatic conversion from that to Long.

Reply:
yeah I was just mentioning that you can remove the type `ApplicationConfigValue?` from the variable declaration, as the type system should be able to correctly infer the type.